packages/react-jss/package.json

```diff
@@ -36,7 +36,7 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "peerDependencies": {
-    "react": ">=0.14"
+    "react": ">=0.15"
```

Comment:
Is this correct? I think it should be `>=15` as React switched to major versions with React v15 (see: https://reactjs.org/blog/2016/04/07/react-v15.html)

Reply:
good catch, yes its true!